From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Wed, 16 Nov 2022 10:07:02 +0100
Subject: chore: patch out partition attribute DCHECK for webviews

https://chromium-review.googlesource.com/c/chromium/src/+/4014994
made it such that <webview>s in Chromium each have their own
StoragePartitionConfig. This is in part because webviews are only
ever instantiated by extensions in Chromium, which is not the case
in Electron.

This change patches it out to prevent the DCHECK.

It can be removed once/if we see a better solution to the problem.

diff --git a/content/browser/site_instance_impl.cc b/content/browser/site_instance_impl.cc
index 4007e92316a6ac59145fa9bc021904fd1b3b0136..afedeaea7fd419f3374ffeebb7ee931219a90f93 100644
--- a/content/browser/site_instance_impl.cc
+++ b/content/browser/site_instance_impl.cc
@@ -226,7 +226,7 @@ scoped_refptr<SiteInstanceImpl> SiteInstanceImpl::CreateForGuest(
     BrowserContext* browser_context,
     const StoragePartitionConfig& partition_config) {
   DCHECK(browser_context);
-  DCHECK(!partition_config.is_default());
+  // DCHECK(!partition_config.is_default());
 
   auto guest_site_info =
       SiteInfo::CreateForGuest(browser_context, partition_config);
